README.md

```diff
@@ -54,17 +54,20 @@ Navigate into the extension directory and initialize it as a Git repository:
 ```bash
 cd saltext-apache
 git init --initial-branch=main
-pip install pre-commit
+pip install pre-commit (may not need to be done again if in a pre-existing virtual environment)
```

Comment:
**chore (blocking):** make the added text a comment so people can still copy/paste the commands